README.markdown

```diff
@@ -315,10 +315,28 @@ Put this into your `$HOME/.zshrc` to call `nvm use` automatically whenever you e
 ```zsh
 # place this after nvm initialization!
 autoload -U add-zsh-hook
+
 load-nvmrc() {
-  if [[ -f .nvmrc && -r .nvmrc ]]; then
-    nvm use
-  elif [[ $(nvm version) != $(nvm version default)  ]]; then
+  nvmversion=`nvm version`
+  current=`pwd`
+  count=`echo $current | grep -o '/' | wc -l`
+  matchStatus='miss'
+  while [[ $count -ge 0 ]]
```

Comment:
This code can get a lot simpler if you use `nvm_find_nvmrc` which will print out the first `.nvmrc` it finds - it's the same code nvm uses internally.